.npm/pnpm-lock.yaml

```diff
@@ -0,0 +1,1429 @@
+lockfileVersion: '6.0'
```

Comment:
Do we need this lock file? If yes, how can we update it, and do we need to worry about keeping it up to date?

Reply:
Yes, we need it to pin the exact versions of all dependencies.

It will only change when and if we add new dependencies to the `.npm` workspace or choose to update any of the existing ones. Although an auto-update routine could be implemented, it's ill-advised. Better to keep things manually and revisit this every once in a while, if necessary, as we go.

FWIW, as this will be a fundamental piece of the TS SDK from now on, we'll know if something is getting outdated and needs a bump (like node, pnpm, or any other).